transport_nantes/topicblog/urls.py

```diff
@@ -10,137 +10,215 @@
 """
 
 app_name = 'topic_blog'
-urlpatterns = [
-    path('t/<slug:the_slug>/', views.TopicBlogItemView.as_view(),
-         name='view_item_by_slug'),
-    path('e/<slug:the_slug>/', views.TopicBlogEmailView.as_view(),
-         name='view_email_by_slug'),
-    path('p/<slug:the_slug>/', views.TopicBlogPressView.as_view(),
-         name='view_press_by_slug'),
-    path('la/<slug:the_slug>/', views.TopicBlogLauncherView.as_view(),
-         name='view_launcher_by_slug'),
-    path('mlp/<slug:the_slug>/', views.TopicBlogMailingListPitchView.as_view(),
-         name='view_mlp_by_slug'),
 
-    path('e/i/<str:token>', views.beacon_view,
-         name='beacon_view'),
-
-    path('admin/t/view/<int:pkid>/', views.TopicBlogItemViewOne.as_view(),
+topicblogitem_urls = [
```

Comment:
This is a good commit, but why is it in this PR?  This PR is long and has some minor issues, whereas this commit would have just been a simple approve (and so not needed to be re-read on next iteration).

Reply:
This is because I've used the new url list format in this PR, creating a "topicblogpanel_urls" list in the next commit. So we would have to merge this change before TBPanels.

I can open a separate PR, but I thought this was a well isolated commit, enough to be reviewed once and approved without having to look at it twice. :)

Reply:
It would have been approved already if separate, and you would have quietly rebased this one on it so as to use it...